terraform/content-store/cloudmap.tf

```diff
@@ -0,0 +1,14 @@
+resource "aws_service_discovery_service" "service" {
+  name = var.service_name
+
+  dns_config {
+    namespace_id = var.govuk_publishing_platform_namespace_id
+
+    dns_records {
+      ttl  = 60
```

Comment:
We might want to revisit this later (e.g. Amazon [previously suggested 10](https://aws.amazon.com/blogs/aws/amazon-ecs-service-discovery/)) but no need to worry about it for now.

Reply:
Thanks, I've pushed a commit for this.